charts/generic-aws-prometheus-alerts/values.yaml

```diff
@@ -20,7 +20,7 @@ additionalRuleLabels: {}
 #   database-name-id: "Database nice display name"
 #   database-name-2-id: "Database nice display name 2"
 
-rdsAlertsCPUThreshold: 5
+rdsAlertsCPUThreshold: 1
```

Comment:
why these values are low.?

Reply:
To trigger the alerts, it was set to 5. Will revert this once the alert trigger is tested

Reply:
this is generic alert and test alert values should be changed to terraform hmpps-service-catalogue (dev namespace) deployment